csm/cli/support_bundle/csm_bundle_generate.py

```diff
@@ -36,14 +41,41 @@ async def init(command):
         :return:
         """
         # Read Config to Fetch Log File Path
-        log_directory_path = Conf.get(const.CSM_GLOBAL_INDEX, "Log.log_path")
+        csm_log_directory_path = Conf.get(const.CSM_GLOBAL_INDEX, "Log.log_path")
+        alerts_file_path = Conf.get(const.CSM_GLOBAL_INDEX, "SUPPORT_BUNDLE.alerts_file_path")
+        # Fetch alerts for support bundle.
```

Comment:
Fetching Alerts should be done only when component selected will be alert

Reply:
Done